maestro-orchestra/src/main/java/maestro/orchestra/workspace/WorkspaceExecutionPlanner.kt

```diff
@@ -81,8 +96,37 @@ object WorkspaceExecutionPlanner {
                 .takeIf { it.exists() }
     }
 
+    private fun parseFileName(file: Path): String {
+        return file.fileName.toString().substringBeforeLast(".")
+    }
+
+    private fun getFlowsToRunInSequence(
```

Comment:
In cloud, we also check if the list has duplicated items, but I think it's fine to not have that locally